atmos_validation/validate_netcdf/tests/test_final_reports_validation.py

```diff
@@ -6,34 +6,51 @@
 
 
 def test_final_reports_ok():
-    """Tests when final_reports is valid (a lists of string)"""
+    """Tests when final_reports is valid (a lists of string with correct extension)"""
     with xr.open_dataset(PATH_TO_TEST_DATA) as ds:
-        ds.attrs["final_reports"] = ["foo", "bar"]
+        ds.attrs["final_reports"] = ["foo.pdf", "bar.docx"]
         errors = final_reports_validator(ds)
         assert len(errors) == 0
 
 
 def test_final_reports_empty_list_ok():
-    """Tests when final_reports is valid (and empty list)"""
+    """Tests when final_reports is valid (an empty list)"""
     with xr.open_dataset(PATH_TO_TEST_DATA) as ds:
         ds.attrs["final_reports"] = []
         errors = final_reports_validator(ds)
         assert len(errors) == 0
 
 
-def test_final_reports_not_list_not_ok():
-    """Tests when final_reports is not valid (not a lists of string)"""
+def test_final_reports_empty_string_ok():
+    """Tests when final_reports is valid (an empty list)"""
     with xr.open_dataset(PATH_TO_TEST_DATA) as ds:
-        ds.attrs["final_reports"] = "this should be a list of strings"
+        ds.attrs["final_reports"] = ""
         errors = final_reports_validator(ds)
-        assert len(errors) == 1
-        assert "must be a list of strings" in errors[0]
+        assert len(errors) == 0
 
 
 def test_final_reports_list_of_non_strings_not_ok():
-    """Tests when final_reports is not valid (a lists of not all string)"""
+    """Tests when final_reports is not valid (not a string)"""
     with xr.open_dataset(PATH_TO_TEST_DATA) as ds:
-        ds.attrs["final_reports"] = ["this should be a list of strings", 3.14]
+        ds.attrs["final_reports"] = ["some_report.pdf", 1]  # wrong type
         errors = final_reports_validator(ds)
         assert len(errors) == 1
-        assert "must be a list of strings" in errors[0]
+        assert "is not comma-separated string or string list" in errors[0]
+
+
+def test_final_reports_not_string_not_ok():
+    """Tests when final_reports is not valid (not a string)"""
+    with xr.open_dataset(PATH_TO_TEST_DATA) as ds:
+        ds.attrs["final_reports"] = True  # wrong type, should be string
+        errors = final_reports_validator(ds)
+        assert len(errors) == 1
+        assert "is not comma-separated string or string list" in errors[0]
+
+
+def test_final_reports_list_wrong_file_extension_not_ok():
+    """Tests when final_reports is not valid (a lists of not all string)"""
```

Comment:
consider changing the comment in the doc string (a lists of not all string) to something more like (comma separated list with wrong extension)